src/Util/DOIutil.php

```diff
@@ -306,6 +306,7 @@ public function deleteDOI(string $doi)
                 'inactive',
                 '2019',
                 'none supplied',
+                [],
```

Comment:
Are these the empty options/publications?

Reply:
Yes, it expects publications, because the dataset is deleted sending a empty array